test/writing/test_minimize_write.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from policy_sentry.writing.sid_group import SidGroup
+
+
```

Comment:
why a new file
there is already a test file for minimize https://github.com/salesforce/policy_sentry/blob/master/policy_sentry/writing/minimize.py

Reply:
+1. It should be in the existing test file